### PR TITLE
FA-87-SETUP-MSAL

### PR DESCRIPTION
**Pull request by ramirezmorac2**
Merging this PR to log into your account

### CODE REVIEW

File: frontend/src/components/PaymentGateway/PaymentGateway.tsx [hunk 1]
```diff
@@ -16,8 +16,9 @@ const fetchApiKey = async () => {
 export const SubscriptionPlans: React.FC<{ stripePromise: Promise<Stripe | null> }> = ({ stripePromise }) => {
     const { user, organization } = useAppContext();
 
-    const [plans, setPlans] = useState<any[]>([]);
-    const currentPlan = organization?.subscriptionId ? 1 : 0;
+    const [currentPlan, setCurrentPlan] = useState(organization.subscriptionId ? 1 : 0);
```
Comment (namtran6701) [inline frontend/src/components/PaymentGateway/PaymentGateway.tsx:19]: we may have more than two plans in the future, consider this 

`enum PlanStatus {
    NoSubscription = 0,
    Subscribed = 1
}

const [currentPlan, setCurrentPlan] = useState<PlanStatus>(organization.subscriptionId ? PlanStatus.Subscribed : PlanStatus.NoSubscription);
`

Comment (maacaro) [reply frontend/src/components/PaymentGateway/PaymentGateway.tsx]: @ramirezmorac2 take a look of that

Comment (maacaro) [reply frontend/src/components/PaymentGateway/PaymentGateway.tsx]: @namtran6701 I remove that code is no longer need the plans we can getting from the product prices on stripe

Comment (namtran6701) [reply frontend/src/components/PaymentGateway/PaymentGateway.tsx]: Understood!

**Pull request merged**